src/components/Browse/Browse.tsx

```diff
@@ -12,11 +12,13 @@ const Browse = () => {
     // // on mount:
     // React.useEffect( () => getAllDatasets(setDsList), []);
 
+    const dsList = searchResults[0].hits;
```

Comment:
Do we need to have the 0 hardcoded here? or is there a possibility of iterating through the searchResults?

Reply:
It's a thing to ask @deepakunni3, since we always get a single search result JSON with a single 'search result' object definition as far as I know. I don't think it would make sense to iterate through it.

Reply:
Lets get back to this point later, for now lets go with the hardcoded 0